docker/.env

```
@@ -151,23 +156,23 @@ SMQ_USERS_DB_SSL_ROOT_CERT=
-SMQ_EMAIL_HOST=smtp.mailtrap.io
-SMQ_EMAIL_PORT=2525
-SMQ_EMAIL_USERNAME=18bf7f70705139
-SMQ_EMAIL_PASSWORD=2b0d302e775b1e
-SMQ_EMAIL_FROM_ADDRESS=from@example.com
-SMQ_EMAIL_FROM_NAME=Example
+SMQ_EMAIL_HOST=__SMQ_EMAIL_HOST__
+SMQ_EMAIL_PORT=__SMQ_EMAIL_PORT__
+SMQ_EMAIL_USERNAME=__SMQ_EMAIL_USERNAME__
+SMQ_EMAIL_PASSWORD=__SMQ_EMAIL_PASSWORD__
+SMQ_EMAIL_FROM_ADDRESS=__SMQ_EMAIL_FROM_ADDRESS__
+SMQ_EMAIL_FROM_NAME=Cube AI
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMQ_EMAIL_FROM_NAME key should go before the SMQ_EMAIL_HOST key

(UnorderedKey)